src/System.Runtime.Serialization.Json/tests/System.Runtime.Serialization.Json.Tests.csproj

```diff
@@ -14,7 +14,9 @@
   <ItemGroup>
     <Compile Include="..\..\System.Runtime.Serialization.Xml\tests\Utils.cs" />
     <Compile Include="..\..\System.Runtime.Serialization.Xml\tests\SerializationTypes.cs" />
+    <Compile Include="..\..\System.Runtime.Serialization.Xml\tests\SerializationTypes.CoreCLR.cs" />
```

Comment:
Just in case you are not aware of, code file naming conventions can be found from [here](https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/project-guidelines.md).

netcore50 or netcore50aot are package target framework used by NuGet for NetNative, so using NetCore.cs could be confusing as our intention is these tests are _not_ for NetNative.